catalog/files/impl/src/main/java/org/projectnessie/catalog/files/adls/AdlsFileSystemOptions.java

```diff
@@ -35,6 +39,30 @@ public interface AdlsFileSystemOptions {
   /** SAS token to access the ADLS file system. */
   Optional<KeySecret> sasToken();
 
+  /**
+   * Enable short-lived user-delegation SAS tokens per file-system.
+   *
+   * <p>The current default is to not enable short-lived and scoped-down credentials, but the
+   * default may change to enable in the future.
+   */
+  Optional<Boolean> userDelegationEnable();
+
+  /**
+   * Expiration time / validity duration of the user-delegation <em>key</em>, this key is
+   * <em>not</em> passed to the client.
+   *
+   * <p>Defaults to 7 days minus 1 minute (the maximum), must be >= 1 second.
```

Comment:
Why do we need a long key expiry default? It looks like we discard the key after obtaining the SAS token? :thinking: